test/ecto/changeset_test.exs

```diff
@@ -118,6 +127,37 @@ defmodule Ecto.ChangesetTest do
     refute changeset.valid?
   end
 
+  test "cast/4: empty parameters are passed to embeds" do
+    changeset = cast(%Post{}, :empty, ~w(authors))
+    assert changeset.changes.authors == []
+
+    changeset = cast(%Post{authors: [%Author{}]}, :empty, ~w(authors))
+    [author_changeset] = changeset.changes.authors
+    assert author_changeset.model == %Author{}
+    assert author_changeset.params == nil
+    assert author_changeset.changes == %{}
+    assert author_changeset.errors == []
+    assert author_changeset.validations == []
+    assert author_changeset.required == [:name]
+    assert author_changeset.optional == []
+    assert author_changeset.action == :update
```

Comment:
There no params... so there isn't anything really we can do... I am thinking that, if it is an embeds_many, we should just create a single empty changeset in the list (with no action) and that is it.